styles/components/Internship/CommitteeInternshipInfo.scss

```diff
@@ -0,0 +1,79 @@
+.committee-internship-info {
+  background-color: #FFF;
+  border-radius: 1rem;
+  box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.3), 0 0.0625rem 0.125rem rgba(0, 0, 0, 0.2);
+  font-size: 1.25em;
+  margin: auto;
+  margin-bottom: 50px;
+  padding: 1.125em 1.5em;
+  position: relative;
+  width: 80vw;
+  
+  &.studio {
```

Comment:
Can we do this programatically with a SASS loop / mixin instead of duplicating the code like this?

Reply:
the reason i didn't use a SASS loop is because the color is labelled differently for each committee. Not sure how I'd use mixins either because of the unique color name for each committee.